src/main.rs

```diff
@@ -18,7 +18,12 @@ fn main() -> Result<(), Error> {
             Arg::with_name("v")
                 .short("v")
                 .multiple(true)
-                .help("Verbosity, add more for more verbose output."),
+                .help("Increases the verbosity"),
+        )
+        .arg(
+            Arg::with_name("proxy")
+                .long("proxy")
+                .help("Enables PROXY protocol when running in ws_to_tcp mode"),
```

Comment:
Are there downsides to defaulting this to be enabled?

Reply:
Yea, so the destination tcp server needs to also support the PROXY protocol because the first thing that will get sent over the new connection is the PROXY header. If you try to send the proxy header to a TCP server that doesn’t understand it, likely it will just cause the connection to break.

Our use case is that the TCP server uses the IP address to Geolocate the client and return a country code to be displayed as a flag next to the players name. However, when the server is sitting behind the websocket bridge, all ip addresses seen by the server are the internal address of the docker network interface, so we need the websocket bridge to pass along the real IP. And since the websocket bridge is also behind CloudFlare it doesn’t even know the real IP address and needs to read it from the `X-Forwarded-For` header.

Reply:
:+1:  I see! Thanks for the explanation, definitely doesn't make sense to default this to enabled then.